pegasus/sites.v3/code.org/public/about/2021.md.partial

```diff
@@ -293,395 +294,259 @@ Code.org® is dedicated to the vision that every student in every school should
-      <tr>
-        <td>Hack the Box</td>
-        <td>Jonathan Gayek</td>
-      </tr>
-      <tr>
-        <td>Mitchell International</td>
-        <td>Forrest Jordan</td>
-      </tr>
-      <tr>
-        <td>Munich Re Digital Partners</td>
-        <td>Edmond Lau</td>
-      </tr>
-      <tr>
-        <td>Ping Identity</td>
-        <td>David and Angela Longo</td>
-      </tr>
-      <tr>
-        <td>Quanex Foundation</td>
-        <td>Peri McDonald</td>
-      </tr>
-      <tr>
-        <td>The Red Hat Charitable Giving Fund of Triangle Community Foundation</td>
-        <td>Mr. Greg Isao Nakamoto and Mrs. Lauri Bortscheller Nakamoto</td>
-      </tr>
-      <tr>
-        <td>Stripe</td>
-        <td>Pratt Family Fund</td>
-      </tr>
-      <tr>
-        <td>Twitch Interactive Inc.</td>
-        <td>Nicholas Schneider</td>
-      </tr>
-      <tr>
-        <td>The Wonderful Company Foundation</td>
-        <td>Kirsten Schreiter</td>
-      </tr>
-      <tr>
-        <td>&nbsp;</td>
-        <td>Danny Sheridan</td>
-      </tr>
-      <tr>
-        <td>&nbsp;</td>
-        <td>Lawrence Tanimoto and Connie Guan</td>
-      </tr>
-      <tr>
-        <td>&nbsp;</td>
-        <td>Molly Tonnell</td>
-      </tr>
-      <tr>
-        <td>&nbsp;</td>
-        <td>Matthew Walker</td>
-      </tr>
-      <tr>
-        <td>&nbsp;</td>
-        <td>Jason Weill</td>
-      </tr>
-      <tr>
-        <td>&nbsp;</td>
-        <td>Kristin Yeoman</td>
-      </tr>
-      <tr>
-        <td>&nbsp;</td>
-        <td>David Youngworth</td>
-      </tr>
-      <tr>
-        <td>&nbsp;</td>
-        <td>Anonymous (4)</td>
-      </tr>
-    </tbody>
-  </table>
+<section class="flex-container">
+  <img src="/images/fit-150/avatars/amazon.jpg" alt="Amazon logo" />
+  <img src="/images/fit-150/avatars/microsoft.jpg" alt="Microsoft logo" />
+  <p><strong>Anonymous</strong></p>
+</section>
+
+<h2>Gold Supporters ($1,000,000+)</h2>
+<section class="flex-container logos">
+  <img src="/images/fit-150/avatars/ballmer_group.jpg" alt="Ballmer Group logo" />
+  <img src="/images/fit-150/avatars/google.png" alt="Google logo" />
+</section>
+<section class="flex-container text">
+  <p><strong>Charlie Lee</strong></p>
+  <p><strong>Anonymous</strong></p>
+</section>
+
+<section class="lists no-flex no-padding">
+  <section class="flex-container no-padding mobile-hide">
+    <h3>Institutional Supporters</h3>
+    <h3>Individual Supporters</h3>
+  </section>
+
+  <h4>$500,000+</h4>
+  <div class="col-50">
+    <h5 class="mobile-show">Institutional Supporters</h5>
+    <ul>
+      <li>Atlassian Foundation International</li>
+      <li>Benevity</li>
+      <li>Cisco Systems Foundation</li>
+      <li>Infosys Foundation USA</li>
+      <li>Pluralsight</li>
+    </ul>
+  </div>
+  <div class="col-50">
+    <h5 class="mobile-show">Individual Supporters</h5>
+    <ul>
+      <li>Rebecca and Alfred Lin</li>
+    </ul>
+  </div>
+  <div class="clearfix"></div>
+
+  <h4>$250,000+</h4>
```

Comment:
Could we generate this from the gsheet instead?  I hadn't realised we had hardcoded these lists anywhere.. the idea was that the gsheet should have all the necessary information to generate them programmatically.

Reply:
I think we could achieve that goal but done programmatically and driven from the same gsheet as our other donor lists.  Is there a backlog we can add this potential work item to?

And if nothing else, we should be able to switch this to markdown rather than HTML.